.circleci/config.yml

```diff
@@ -120,9 +120,9 @@ jobs:
       - image: datadog/docker-library:ansible_debian_2_10
     steps:
       - checkout
-      # TODO: upgrade ansible-lint to latest (will require updating the image for this task)
-      - run: pip install ansible-lint==6.17
-      - run: ansible-lint -v --exclude=.circleci/config.yml .
+      - run: pip install ansible-lint==6.17 galaxy-importer
+      - run: .circleci/collection-structure-setup.sh
+      - run: cd ansible_collections/datadog/dd && ansible-lint -v --profile=production --exclude=galaxy.yml --exclude=meta/
```

Comment:
Could we make all of this part of the script? What I would like to do is just run e.g. `ci-test/test-collection-lint.sh` and get the same result that CI has. We should put the script somewhere else than the `.circleci` repository, because that makes it seem like it's only for CI usage, but it's actually desirable that this file is used by contributors to check their PRs. I propose to move it to `ci-test` and also mention it in `CONTRIBUTING.md`. Does that make sense?

Reply:
Good idea 👍